src/main/java/org/testng/TestNG.java

```diff
@@ -474,13 +474,17 @@ public void setThreadCount(int threadCount) {
   @Deprecated
   public void setParallel(String parallel) {
     if (parallel == null) {
-      setParallel(XmlSuite.ParallelMode.FALSE);
+      setParallel(XmlSuite.ParallelMode.NONE);
     } else {
       setParallel(XmlSuite.ParallelMode.getValidParallel(parallel));
     }
   }
 
   public void setParallel(XmlSuite.ParallelMode parallel) {
+    if (parallel == XmlSuite.ParallelMode.FALSE || parallel == XmlSuite.ParallelMode.TRUE) {
+      Utils.log("TestNG", 1, "[WARN] 'parallel' value 'true' and 'false' are deprecated, use 'none' instead.");
```

Comment:
`'none' will be used instead` instead of `use 'none' instead`? The idea is to warn user that the behavior could not be the expected one.